doc/changelog/06-Ltac2-language/19961-eval-with-zero-goals.rst

```diff
@@ -0,0 +1,3 @@
+- **Fixed:** remove a constraint for ltac2 tactic evaluation requiring
```

Comment:
Sorry to be a bit nitpicky but I find the changelog a bit mysterious. Ltac2 evaluation could also refer to simply running a tactic in a goal in proof mode in general. I'd rather have a shorter but more to-the-point message like "`Ltac2 Eval` does not require to be focussed in a goal anymore."

Reply:
Np. Thanks for feedback.